source/features/extend-diff-expander.tsx

```diff
@@ -6,16 +6,26 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '../feature-manager.js';
 
+const diffSelectors = [
+	'.diff-view .js-expandable-line',
+	'.diff-line-row:has(button[data-direction])',
+];
+
+const diffExpandSelector = [
+	'.js-expand',
+	'button[data-direction]',
+];
```

Comment:
Can you add comments on each? Please follow these instructions on every PR so I don't have to leave these comments on every PR 🙏